region/maridia/outer/Mama Turtle Room.json

```diff
@@ -664,6 +664,24 @@
         "The easiest method is to jump over her by jumping on her babies, or to Morph under her."
       ]
     },
+    {
+      "link": [3, 1],
+      "name": "SpaceJump Escape with Walljump",
+      "requires": [
+        "SpaceJump",
+        "canPreciseWalljump",
+        {"or": [
+          "canHeroShot",
+          "h_canUsePowerBombs",
+          "Spazer",
+          "Wave"
+        ]}
+      ],
+      "note": [
+        "From the safe spot above the item, use a Power Bomb or shoot down the wall to break the item block.",
+        "Spin jump down and collect the item, then walljump into SpaceJump to escape without falling into the water."
+      ]
```

Comment:
The way that the link is set up, this will be satisfiable if the item is one that is needed for the strat. So this could be possible even if the item is Space, Wall Jump boots, PBs, Morph, Wave, or Spazer. This can be solved cleanly with another node, but that may not be necessary. Looking through all of these to see if it is necessary:
- This hero shot is quite easy relative to a normal hero shot, so I think it's probably fine if its any of the last 4 and you are just forced to do the hero shot even if it's toggled off. (Actually, the other strats here don't require the canHeroShot and just assume it is free.)
- If you have space jump, you can always get to the top right ledge, grab the item while in a spin and wall jump back up, so it should be fine if its wall jump boots.
- It looks like `Reveal Item while Jumping` is the only strat that can get to the item and not be in spin pose, but if you can wall jump, you could go to the top first.

So, I guess the TL;DR is that this method of representing it should be fine, but could be more error prone if incompatible strats are added that get to 3. 
```suggestion
      ],
      "devNote": [
        "If one of the items that is needed to perform this strat is in this location, the strat could be slightly harder.",
        "Samus should always be able to get here in a spin pose or to the ledge above regardless of which item is in this location."
      ]
```

Reply:
I'm confused... I thought the item being acquired was _never_ considered in the routing/reachability?  IE, you have to have Space and one of Spazer, Wave, PB, or Hero Shot in advance to acquire whatever is here, and the item content here is irrelevant to reachability.

Reply:
Or wait, my mental model is Maprando specific... maybe you're thinking at a higher level about how this data might be used in other contexts?

If that's the case, then you're right.  Again it's beyond my level of comfort, but I think remodeling this room with a "safe" node above 3 is probably long term better.  But we also need a way to model "spinning" state... reaching node 3 and collecting the item while spinning vs. not spinning is what actually requires the hero shot / wave / spazer / pb.  And then escaping from 3 with walljump+spacejump is a separate node transition.

Hypothetically speaking